src/test/scala/is/hail/methods/LDPruneSuite.scala

```diff
@@ -30,7 +30,7 @@ class LDPruneSuite extends SparkSuite {
     new MultiArray2(nVariants, nVariants, r2.toArray)
   }
 
-  def isUncorrelated(vds: VariantDataset, r2Threshold: Double, windowSize: Int): Boolean = {
+  def isUncorrelated(vds: VariantSampleMatrix, r2Threshold: Double, windowSize: Int): Boolean = {
```

Comment:
should we change the corresponding variable names? To like `vsm` or something

Reply:
VSM is going to be renamed, see: http://dev.hail.is/t/proposed-changes-to-python-object-hierarchy/36/2.  I was going to wait until after that happens.  (Also, I feel this will be pervasive and tedious, but low value so I'm in particular hurry.)